File: dev-superior-algoritmo-e-logica-de-programacao/java/src/exemplos/SaidaDeDados.java
package exemplos;

import java.util.Locale;

public class SaidaDeDados {

    public static void main(String[] args) {

        Locale.setDefault(Locale.US);

        System.out.print("Bom dia");
        System.out.print("Boa noite");

        System.out.println("Bom dia");
        System.out.println("Boa noite");

        int a, b;
        a = 10;
        b = 20;
        System.out.println(a);
        System.out.println(b);

        double c;
        c = 2.3456;
        System.out.println(String.format("%.2f", c));

        int idade;
        double salario;
        String nome;
        char sexo;

        idade = 32;
        salario = 4560.9;
        nome = "Maria Silva";
        sexo = 'F';

        System.out.println("A funcionaria " + nome + ", sexo " + sexo +
                ", ganha " + String.format("%.2f", salario) + " e tem "
                + idade + " anos");

    }
}

File: nelio-alves/algoritmo-e-logica-de-programacao/python/exemplos/saida_de_dados.py
print("bom dia", end="")
print("boa noite", end="")

print("bom dia")
print("boa noite")

a: int; b: int
a = 10
b = 20
print(a)
print(b)

c: float
c = 2.3456
print("{:.2f}".format(c))

idade: int
salario: float
nome: str
sexo: str

idade = 32
salario = 4560.9
nome = "Maria Silva"
sexo = "F"

print(f"A funcionaria {nome}, sexo {sexo}, ganha {salario:.2f} e tem {idade} anos")

print("A funcionaria {:s}, sexo {:s}, ganha {:.2f} e tem {:d} anos".format(nome, sexo, salario, idade))